devtools/cli/src/main/java/io/quarkus/cli/update/RewriteGroup.java

```diff
@@ -5,11 +5,12 @@
 public class RewriteGroup {
 
     @CommandLine.Option(order = 0, names = {
-            "--no-rewrite" }, description = "Disable the rewrite feature.", defaultValue = "false")
```

Comment:
should we keep this as hiden but backwards compatible?

Reply:
Kept